### DISCUSSION TITLE
posts/tutorial4/

### DISCUSSION THREAD

**Discussion by giscus** [7w1/7w1.github.io#2] [Announcements]
# posts/tutorial4/

Today, we’re going to begin our first Survey Ship, but first…

https://7w1.github.io/posts/tutorial4/

**Comment by Khaligufzel** [comment]
Hi hello!
Thank you for this wonderful tutorial!
This tut is the best one available, so thank you a lot!

I have an issue though- I believe I have newer version of Aurora- I can't convert CI to anything, these options are just not there. Any suggestions? :)

**Reply by Khaligufzel** [reply]
I figured it out, only the names changed! There is no more "Convert (...)" :)
Sorry!
